app/config/FrontendAppConfig.scala

```diff
@@ -40,6 +41,10 @@ class FrontendAppConfig @Inject()(configuration: Configuration) {
   private val declareArrivalNotificationStartUrl = configuration.get[String]("declare-transit-movement-arrival-frontend.startUrl")
   val declareArrivalNotificationUrl              = s"$declareArrivalNotificationRoute/$declareArrivalNotificationStartUrl"
 
+  private val declareUnloadingRemarksRoute         = configuration.get[String]("declare-transit-movement-unloading-frontend.host")
+  private val declareUnloadingRemarksStartUrl      = configuration.get[String]("declare-transit-movement-unloading-frontend.startUrl")
+  val declareUnloadingRemarksUrl: String => String = (mrn => s"$declareUnloadingRemarksRoute/$declareUnloadingRemarksStartUrl/$mrn/unloading-guidance")
```

Comment:
Brackets aren't needed

Reply:
If we are going on this route can we not get baseUrl rather than Route and StartUrl as this is looking clunky?

Reply:
Removed